process/archiving.md

```diff
@@ -0,0 +1,35 @@
+# CNCF Project Archiving Process v1.0
+
+Open source projects have a lifecycle and there are times that projects become inactive due to a variety of reasons. There are also cases where a project may no longer want to be supported by the TOC.
+
+## Archiving Criteria
+
+There are different criteria to consider when archiving a project, but here are ones that the TOC looks for:
```

Comment:
"...here are some that..."

Reply:
Similarly I’m not sure we really need to specify archiving criteria - it could be as simple as the super-majority of the TOC no longer consider that the project meets the qualification criteria.